packages/web-components/src/components/combo-button/combo-button.scss

```
@@ -0,0 +1,23 @@
+//
+// Copyright IBM Corp. 2025, 2025
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+$css--plex: true !default;
+@use '@carbon/styles/scss/config' as *;
+@use '@carbon/styles/scss/motion' as *;
+@use '@carbon/styles/scss/components/combo-button/index' as *;
+
+:host {
```

this seems to work, but we usually always put the component in the host

```suggestion
:host(#{$prefix}-combo-button) {
```